PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -372,7 +372,10 @@ VerifyFwVersion (
     Status = GetComponentInfoByPartition(FLASH_MAP_SIG_STAGE1A, FALSE, &CompBase, &CompSize);
   } else if (FwPolicy.Fields.UpdatePartitionA == 0x1) {
     Status = GetComponentInfoByPartition(FLASH_MAP_SIG_STAGE1A, TRUE, &CompBase, &CompSize);
+  } else {
+    Status = EFI_INVALID_PARAMETER;
```

Comment:
For consistency, why not just initialize Status to  EFI_INVALID_PARAMETER at line 370 before the if block?